Dockerfile

```diff
@@ -53,4 +53,6 @@ RUN backend="$(ls -1 lib/ | grep ex_backend)" && \
     rm -rf lib/$backend/priv/static/ && \
     mv static/ lib/$backend/priv/
 
+HEALTHCHECK --interval=30s --start-period=2s --retries=2 --timeout=3s CMD curl -v --silent --fail http://localhost:8080/ || exit 1
```

Comment:
Is it possible to get port from configuration ?
It's currently specific to a configuration ...

Reply:
Done.
When you build docker image you could pass the custom as follow:
`docker build -t mediacloudai/ex_backend:1.0.0 --build-arg customAppPort=9080 .`